### ISSUE TITLE
`pip install -e .` does not work in a new Python environment

### ISSUE THREAD

**Issue by romanwerpachowski** [PrimerAI/blanc#36]
When I try to install blanc in a new Python environment using `pip install -e .`, this fails because the installation process requires nltk and torch to be present during installation.

**Comment by romanwerpachowski** [comment]
It's possibly because of this line in setup.py:

from blanc.__version__ import __version__

which triggers importing many other blanc modules.

**Comment by OlegVasilyev4096** [comment]
Thanks! I will look at this anyways, appreciate if you could give the exact error message you got.

**Comment by romanwerpachowski** [comment]
Running `pip install -e .` in a fresh Python virtual environment:

```
Obtaining file:///C:/Users/romanwe/source/repos/blanc
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\romanwe\source\repos\blanc\.env\scripts\python.exe' -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\romanwe\\source\\repos\\blanc\\setup.py'"'"'; __file__='"'"'C:\\Users\\romanwe\\source\\repos\\blanc\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\romanwe\AppData\Local\Temp\pip-pip-egg-info-eoax_6ln'
         cwd: C:\Users\romanwe\source\repos\blanc\
    Complete output (9 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\romanwe\source\repos\blanc\setup.py", line 3, in <module>
        from blanc.__version__ import __version__
      File "C:\Users\romanwe\source\repos\blanc\blanc\__init__.py", line 2, in <module>
        from .blanc import BlancHelp, BlancTune
      File "C:\Users\romanwe\source\repos\blanc\blanc\blanc.py", line 7, in <module>
        from nltk.tokenize import sent_tokenize
    ModuleNotFoundError: No module named 'nltk'
    ----------------------------------------
```

**Comment by romanwerpachowski** [comment]
Suggested fix in #37.
